tests/trigonometry.cpp

```diff
@@ -31,6 +35,19 @@ TEST(trigonometry, cos)
         auto cos_fixed = static_cast<double>(cos(P(flt_angle)));
         EXPECT_TRUE(HasMaximumError(cos_fixed, cos_real, MAX_ERROR_PERC));
     }
+
+    // Boundary-value analysis
+    constexpr int32_t raw_values[] = {INT32_MIN, 2147380703, 2147380704, INT32_MAX};
```

Comment:
Could you add a comment why these two "special" values were chosen?
(same thing in the other test).